### DISCUSSION TITLE
is MapBox11 available for Fabric ?

### DISCUSSION THREAD

**Discussion by erwanlpfr** [rnmapbox/maps#3491] [General]
Hello ! Firstly thank you for your work !

I have actually a question, since React Native released the Bridgless mode, I tried to run my apps with the new arch activated.
It turns out that my setup Mapbox v11 crashed.
I isolated mapbox as the cause of the crash. It was actually not that obvious since the crash was silent and in the native logs on iOS and Android it was a crash in the JS engine.

So I checked and I see that in GH Action there is no test for mapbox 11 fabric, so does my guess right ?
If not, maybe I can dig more to find more logs !

It crashes in 3 completely different setups of Mapbox v11. I did not try with V10 tho.

Thank you !

**Comment by mfazekas** [comment]
@erwanlpfr it should work. But in general fabric is less used, so there will be more issues there. Please report concrete issues
